src/rdbms/mongoose_rdbms.erl

```diff
@@ -232,33 +239,62 @@ prepared(Name) ->
 
 -spec execute(HostType :: server(), Name :: atom(), Parameters :: [term()]) ->
                      query_result().
-execute(HostType, Name, Parameters) when is_atom(Name), is_list(Parameters) ->
-    sql_call(HostType, {sql_execute, Name, Parameters}).
+execute(HostType, Name, Parameters) ->
+    execute(HostType, ?DEFAULT_POOL_NAME, Name, Parameters).
+
+-spec execute(HostType :: server(), PoolName :: atom(), Name :: atom(), Parameters :: [term()]) ->
```

Comment:
What we call `Name` in this module, is called `Tag` in `mongoose_wpool`, and there is a type `mongoose_wpool:tag()` as well. So for me it should be:

```suggestion
-spec execute(HostType :: server(), PoolName :: atom(), Tag :: mongoose_wpool:tag(), Parameters :: [term()]) ->
```

This way it would be more consistent.